### ISSUE TITLE
Last two second level headers in a long list hidden by "Author" "Date" in the sidebar dropdown

### ISSUE THREAD

**Issue by ltrainstg** [juba/rmdformats#69]
# Problem:  End of the sidebar hidden
ReadTheDown format does not show all second level headers in the sidebar dropdown if there are too many, ~30+, items in the sidebar and the Author Date is asked for in the YAML. When there is no author or date it looks fine. 

## Ideal Solution
This section should collapse when we scroll all the way to the bottom or it should have some way to collapse this section. 

# Details

## Image Below: 
![image](https://user-images.githubusercontent.com/16125810/76130218-46d95400-5fc7-11ea-8263-405cf1e41f0c.png)

## Code to reproduce:
````

---
title: "Looking at Top Reported Outcomes"
author: "Lionel Duarte"
date: "`r Sys.Date()`"
output:
  rmdformats::readthedown:
    code_folding: hide
    gallery: yes
    lightbox: yes
    self_contained: yes
    thumbnails: yes
    use_bookdown: no
  html_document:
    df_print: paged
editor_options:
  chunk_output_type: console
---


```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)

fillerText <- "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."

```

# Main Header

## 1
`r fillerText`

## 2

`r fillerText`

## 3

`r fillerText`

## 4

`r fillerText`

## 5

`r fillerText`

## 6

`r fillerText`

## 7

`r fillerText`

## 8

`r fillerText`

## 9

`r fillerText`

## 10

`r fillerText`

## 11

`r fillerText`

## 12

`r fillerText`

## 13

`r fillerText`

## 14

`r fillerText`

## 15

`r fillerText`

## 16

`r fillerText`

## 17

`r fillerText`

## 18

`r fillerText`

## 19

`r fillerText`

## 20

`r fillerText`

## 21


`r fillerText`

## 22

`r fillerText`

## 23

`r fillerText`

## 24

`r fillerText`

## 25

`r fillerText`

## 26

`r fillerText`

## 27

`r fillerText`


## 28

`r fillerText`


## 29

`r fillerText`


## 30

`r fillerText`


# Second Header

````

**Comment by juba** [comment]
Could you try with the development version ? I think this has already been fixed in #66...

Thanks for taking the time to report the problem with a reproducible example.

**Comment by ltrainstg** [comment]
Yes. That does it thanks.
